packages/scripts/network-setup/deployNftLaunchpad.ts

```diff
@@ -0,0 +1,168 @@
+import { program } from "commander";
+import { cloneDeep } from "lodash";
+import os from "os";
+import path from "path";
+
+import { InstantiateMsg as NftLaunchpadInstantiateMsg } from "@/contracts-clients/nft-launchpad";
+import {
+  allNetworks,
+  CosmosNetworkInfo,
+  getNetworkFeature,
+  NetworkFeature,
+} from "@/networks";
+import { CosmWasmLaunchpad } from "@/networks/features";
+import {
+  DeployOpts,
+  initDeploy,
+  instantiateContract,
+  storeWASM,
+} from "@/scripts/network-setup/deployLib";
+
+export const deployNftLaunchpad = async ({
```

Comment:
don't export things from a file with a `main()` execution, otherwise this main will be executed when you import these things
```suggestion
const deployNftLaunchpad = async ({
```

Reply:
Done